package.json

```diff
@@ -7,6 +7,7 @@
   "author": "Andrew Dassonville <dassonville.andrew@gmail.com>",
   "license": "MIT",
   "dependencies": {
+    "cherio": "^1.0.0-rc.2",
```

Comment:
Luckily the person maintaining `cherio` is aliasing it to the correct dependency, `cheerio`, but we should be using the official name of this dependency: https://www.npmjs.com/package/cheerio